app/services/foreman_ansible/proxy_load_balancer.rb

```diff
@@ -0,0 +1,36 @@
+module ForemanAnsible
+  class ProxyLoadBalancer
```

Comment:
This class is literally taken from RemoteExecution, move it into a gem then we can use it here?